client/components/userInput.jsx

```diff
@@ -1,13 +1,76 @@
-import React from 'react';
+import React, { useState } from 'react';
 
 export default function UserInput() {
+  const [generateOptions, setGenerateOptions] = useState({
```

Comment:
So this is a reasonable way to write this code. I see below, though, that you only set and read the items individually. So there's not really any benefit to having them both in one object.

As an alternative, you could just have 2 state variables: `prompt` and `size`.

This is optional, of course, but it might make your code simpler.